doc/usermanual/lighttable/concepts/ratings_color.xml

```diff
@@ -28,14 +28,14 @@
     <xref linkend="gui_options"/>). You can later revise the rating at any time. You can also
     mark an image as <quote>rejected</quote> by pressing the
     <inlinegraphic fileref="&icon_lighttable_reject;" scalefit="1" width="2%" align="center"/>
-    icon or typing the <emphasis>r</emphasis> key. This will remove all stars. You can reverse
+    icon or typing the <emphasis>R</emphasis> key. This will remove all stars. You can reverse
```

Comment:
I am not sure about capitalizing letter keys. That might make people think they have to press shift, too.

Reply:
In the whole books the letters of keyboard shortcuts are always written in capital letters. In addition, these are also shown as uppercase letters on the keyboard. Here are a some examples:
Windows: https://support.microsoft.com/en-ca/help/12445/windows-keyboard-shortcuts
Gnome: https://wiki.gnome.org/Design/OS/KeyboardShortcuts
KDE:   https://docs.kde.org/trunk5/en/applications/fundamentals/kbd.html
MacOS: https://support.apple.com/en-ca/HT201236`

Cross Platform documentation example: Adobe: https://helpx.adobe.com/photoshop/using/select-mask.html

Reply:
Today I saw that in the darktable settings the shortcut keys for single keys are also capitalized. (german darktable version)

Reply:
Good point. Let's have them as capital letters then.